test/orfs/gcd/BUILD

```diff
@@ -12,6 +12,9 @@ orfs_flow(
         "DIE_AREA": "0 0 16.2 16.2",
         "PLACE_DENSITY": "0.35",
         "OPENROAD_HIERARCHICAL": "1",
+        # Demonstrate clock period specified in arguments, useful
+        # for Optuna scripts.
+        "ABC_CLOCK_PERIOD_IN_PS": "310.001",
```

Comment:
Curious why you went for 310.001 and not 310

Reply:
Same number as it used to be, but textually different, so I was in no doubt it is working and I don't have to redo metadata for tests.